app/__init__.py

```diff
@@ -14,15 +14,11 @@
 def create_app(testing=False):
     app = Flask(__name__, static_url_path="", static_folder="static")
 
-    if testing:
-        app.config["TESTING"] = True
-        app.config["SQLALCHEMY_DATABASE_URI"] = "sqlite:///:memory:"
-        app.config["SQLALCHEMY_TRACK_MODIFICATIONS"] = False
-    else:
-        app.config["SQLALCHEMY_DATABASE_URI"] = os.getenv("DATABASE_URI")
-        app.config["SQLALCHEMY_TRACK_MODIFICATIONS"] = False
-        app.config["SECRET_KEY"] = os.urandom(16)
-        Bootstrap(app)
+    app.config["TESTING"] = testing
+    app.config["SQLALCHEMY_DATABASE_URI"] = os.getenv("DATABASE_URI")
+    app.config["SQLALCHEMY_TRACK_MODIFICATIONS"] = False
+    app.config["SECRET_KEY"] = os.urandom(16)
+    Bootstrap(app)
```

Comment:
Do these configs apply to environments other than testing?

Reply:
i figure SQLALCHEMY_DATABASE_URI/DATABASE_URI would apply to other environments. everything else can be moved to the [fixture](https://github.com/GSA/datagov-harvesting-logic/blob/03f52c656e0feeff89a9b47ea42fac1b1c3e98d1/tests/conftest.py#L20)

Reply:
is `SECRET_KEY` something we want to use?

Reply:
`SECRET_KEY` can anything for the local environment; in cloud.gov it should be a cf env variable.

Reply:
set `app.config["TESTING"] = True` in conftest.py and then do conditional logic here based on that.